UnitystationLauncher/ViewModels/InstallationsPanelViewModel.cs

```diff
@@ -92,10 +87,61 @@ private async Task OnAutoRemoveChangedAsync()
             }
         }
 
+        private void InitializeInstallationsList()
+        {
+            Log.Information("Initializing installations list...");
+            List<Installation> installations = _installationService.GetInstallations();
+
+            foreach (Installation installation in installations)
+            {
+                InstallationViews.Add(new(installation, _installationService));
+            }
+
+            Log.Information("Scheduling periodic refresh for installations list...");
+            RxApp.MainThreadScheduler.SchedulePeriodic(_refreshInterval, RefreshInstallationsList);
+        }
+
+        private void RefreshInstallationsList()
+        {
+            List<Installation> installations = _installationService.GetInstallations();
+
+            // Add new
```

Comment:
comments like this and the one on line 119 look like they could be extracted into their own methods and use the name of the method as self-documentation.

Reply:
Moved these to their own methods, both here and in the servers panel